packages/react/toolbar/src/Toolbar.stories.tsx

```diff
@@ -0,0 +1,193 @@
+import * as React from 'react';
+import { Toolbar, ToolbarGroup, ToolbarLabel, ToolbarSeparator, ToolbarItem } from './Toolbar';
+import { css } from '../../../../stitches.config';
+import { ToggleButton } from '@radix-ui/react-toggle-button';
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuArrow,
+} from '@radix-ui/react-dropdown-menu';
+
+import { styledComponents } from '../../menu/src/Menu.stories';
+
+const {
+  StyledRoot: StyledDropDownMenu,
+  StyledItem: StyledDropdownItem,
+  StyledLabel,
+} = styledComponents;
+
+export default { title: 'Components/Toolbar' };
+
+export const Styled = () => {
+  return (
+    <>
+      <ToolbarExample title="Horizontal"></ToolbarExample>
+      <ToolbarExample title="Vertical" orientation="vertical"></ToolbarExample>
+    </>
+  );
+};
+
+const ToolbarExample = ({ title, orientation }: any) => {
+  const anchorRef = React.useRef<HTMLAnchorElement>(null);
+  const inputRef = React.useRef<HTMLInputElement>(null);
+
+  return (
+    <>
+      <h1>{title}</h1>
+      <Toolbar
+        className={StyledToolbar}
+        orientation={orientation}
+        loop={true}
+        aria-label={`${title} toolbar`}
+      >
+        <ToolbarItem className={StyledToolbarItem} as={ToggleButton}>
+          Toggle
+        </ToolbarItem>
+        <ToolbarSeparator className={StyledToolbarSeparator}></ToolbarSeparator>
+        <ToolbarGroup className={StyledToolbarGroup}>
+          <ToolbarLabel as={StyledLabel}>A group</ToolbarLabel>
+          <label className={StyledToolbarItem} htmlFor="input">
+            <span>Input: </span>
+            <ToolbarItem
+              ref={inputRef}
+              id="input"
+              value="42"
+              as={'input'}
+              style={{ maxWidth: 50 }}
+            ></ToolbarItem>
+          </label>
+          <ToolbarItem
+            className={StyledToolbarItem}
+            as={'button'}
+            onKeyDown={(event) => {
```

Comment:
Makes sense, I was trying to keep it generic to make sure it would work with any element, but I have an `onKeyDown` on the anchor.

And you are right, I thought that `composeEventHandlers` require an additional function so I tried to do my own version with `handleClick` then forgot to call `itemProps.onClick()` inside it  :facepalm: